ui/lib/components/conversation/conversation_item.dart

```diff
@@ -1,5 +1,7 @@
 import 'dart:html';
 import 'dart:async';
+import 'package:intl/intl.dart';
+import 'package:katikati_ui_lib/utils/dateTime.dart';
```

Comment:
nit: prefer not to capitalise file names - I think calling the file `datetime.dart` should be fine